<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransitionEditor</class>
 <widget class="QWidget" name="TransitionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Transition Intensity Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="4">
    <widget class="QTableWidget" name="tableTransition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>430</width>
       <height>210</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="buttonAddTransition">
     <property name="toolTip">
      <string>Adds new Gamma</string>
     </property>
     <property name="text">
      <string>Add Gamma</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="buttonQuit">
     <property name="toolTip">
      <string>Exit with no changes to original gammas</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="buttonRemove">
     <property name="text">
      <string>Remove Gamma</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="buttonNormalise">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Renormalise</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelTransition">
     <property name="text">
      <string>Transition  Editor for Level: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelTransitionIntensity">
     <property name="text">
      <string>Summed transition intensity (%):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboBoxIntensity"/>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="buttonMethod">
     <property name="text">
      <string>ApplyModel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
